src/components/atoms/TeamCard/index.jsx

```diff
@@ -0,0 +1,34 @@
+import Body3 from '../../shared/Typography/Body3';
```

Comment:
Do not make individual imports from Typography

Reply:
Done